store/ivy.xml

```diff
@@ -15,13 +15,14 @@
   <dependency org="org.powermock" name="powermock-api-mockito-common" rev="1.6.5"/>
   <dependency org="org.mockito" name="mockito-all" rev="1.10.19" />
   <dependency org="org.javassist" name="javassist" rev="3.18.2-GA" />
-  <dependency org="org.slf4j" name="slf4j-api" rev="1.7.30"/>
-  <dependency org="org.slf4j" name="slf4j-log4j12" rev="1.7.30"/>
+  <dependency org="org.slf4j" name="slf4j-api" rev="1.7.36"/>
```

Comment:
should we move this dependency version at centralized position like we have for ${jetty.version}"?